cron/monthly/user-report.js

```diff
@@ -198,25 +199,25 @@ const processBacker = async FromCollectiveId => {
         createdAt: { [Op.gte]: startDate, [Op.lt]: endDate },
         SubscriptionId: { [Op.ne]: null },
       },
+      deletedAt: null,
     },
     include: [{ model: models.Subscription }],
   });
-  const ordersByCollectiveId = {};
-  orders.map(o => {
-    ordersByCollectiveId[o.CollectiveId] = o;
-  });
+  // group orders(by collective) that either don't have subscription or have active subscription
+  const ordersByCollectiveId = groupBy(orders.filter(o => !o.Subscription || o.Subscription.isActive), 'CollectiveId');
   const collectivesWithOrders = [];
   collectives.map(collective => {
+    // It's only possible to have one order with subscription active Per Collective
+    const collectiveOrder = ordersByCollectiveId[collective.id] && ordersByCollectiveId[collective.id][0];
```

Comment:
What happens if `collectiveOrder` is null?

Reply:
if you look at the `computeStats` method it checks whether the collective has an Order or an expense so we won't have validation problems.

Reply:
@marcogbarcellos ?